app/models/profile.rb

```diff
@@ -14,4 +14,17 @@ class Profile < ActiveRecord::Base
   validates :user, presence: true
   validates_presence_of :first_name, :last_name, :gender, :birth_date, :email, :school, :street_name, :zip_code, :city, :state, :country
 
-end
\ No newline at end of file
+  def age
+    now = Date.today
+    now.year - birth_date.year - (birth_date.to_date.change(:year => now.year) > now ? 1 : 0)
```

Comment:
If a person was born at the 29th February on a leap year and the current year is not a leap year. I think `birth_date.to_date.change(:year => now.year)` will break. Can you add this test case?